hassio/src/dialogs/update/dialog-supervisor-update.ts

```diff
@@ -35,39 +35,23 @@ class DialogSupervisorUpdate extends LitElement {
 
   @internalProperty() private _error?: string;
 
-  @internalProperty() private _name?: string;
-
-  @internalProperty() private _version?: string;
-
-  @internalProperty() private _type?: "core" | "addon";
-
-  @internalProperty() private _updateHandler?: () => Promise<void>;
-
-  @internalProperty() private _snapshotParams?: any;
+  @internalProperty()
+  private _dialogParams?: SupervisorDialogSupervisorUpdateParams;
 
   public async showDialog(
     params: SupervisorDialogSupervisorUpdateParams
   ): Promise<void> {
     this._opened = true;
     this.supervisor = params.supervisor;
```

Comment:
We keep a copy of the supervisor in memory now, we can just use `this._dialogParams.supervisor`?